### DISCUSSION TITLE
how to close arg notice, like "code:" in "return {code: Status::RedisParseErr};"

### DISCUSSION THREAD

**Discussion by chenghua-root** [neoclide/coc.nvim#4912] [Q&A] [answered]
return {code: Status::RedisParseErr};
I want to close the arg notice with "code:"
![image](https://github.com/neoclide/coc.nvim/assets/53387415/67ede54d-771b-407d-bdaf-3821ac5d0713)

**Comment by fannheyward** [comment, marked as answer]
Disable it with `inlayHint.enable`, check `:h coc-inlayHint` and `:h coc-config-inlayHint` for more.

**Reply by chenghua-root** [reply]
worked. thank you
